journalpump/rsyslog.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2019, Aiven, https://aiven.io/
+#
+# This file is under the Apache License, Version 2.0.
+# See the file `LICENSE` for details.
+
+import datetime
+import socket
+import ssl
+
+NILVALUE = "-"
+
+
+def _rfc5424(*, pri, timestamp, hostname, app_id, proc_id, msg_id, msg, sd):
+    data = f"<{pri}>1 {timestamp} {hostname} {app_id} {proc_id} {msg_id}"
+    if sd is not None:
+        data += f" [{sd}]"
+    data += f" {msg}\n"
+    return data.encode("utf-8", "replace")
+
+
+# pylint: disable=unused-argument
```

Comment:
Could name this a bit more verbose than _rfc3164, something like rfc_3164_formatter or somesuch.

Reply:
fixed